app/Models/Behaviors/HasAutoRelated.php

```diff
@@ -22,17 +27,77 @@ public function related($id)
             'exhibitions',
         ];
 
+        // Get all blocks that have this model as a related item
         $relatedBlockItems = Block::whereIn('blockable_type', $blockableTypes)
             ->where('content', 'LIKE', '%"browsers"%')
             ->where('content', 'LIKE', '%' . $modelName . '%')
             ->where('content', 'LIKE', '%' . $id . '%')
             ->get();
 
-        $relatedItems = [];
         foreach ($relatedBlockItems as $relatedBlockItem) {
             $relatedItems[] = $relatedBlockItem->blockable;
         }
 
+        // Get all sidebar items that have this model as a related item
+        $editorialTypes = [
+            'articles',
+            'highlights',
+            'events',
+            'exhibitions',
+            'experiences',
+            'digitalPublications',
+            'videos'
+        ];
+
+        // Set scope to only include sidebar items that can be related to
+        $relatedSidebarItems = DB::table('related')
+            ->where('browser_name', 'sidebar_items')
+            ->where('related_id', $id)
+            ->where('related_type', $modelName)
+            ->whereIn('subject_type', $editorialTypes)
+            ->get();
```

Comment:
Rather than constructing a DB query here, use the A17\Twill\Models\RelatedItem model and the Eloquent methods for retrieving models: https://laravel.com/docs/10.x/eloquent#retrieving-models